pandas/_typing.py

```diff
@@ -24,6 +24,9 @@
 Dtype = Union[str, np.dtype, "ExtensionDtype"]
 FilePathOrBuffer = Union[str, Path, IO[AnyStr]]
 
+# Type for index and columns of DataFrame
+Axes = Iterable[Union["Index", Iterable[str]]]
```

Comment:
The subscript of Iterable here isn't required - can you remove?

Reply:
do you mean remove `str` from the inner `Iterable`?